pkg/kubelet/config/common.go

```diff
@@ -77,7 +77,7 @@ func applyDefaults(pod *api.Pod, source string, isFile bool, hostname string) er
 	// Set the Host field to indicate this pod is scheduled on the current node.
 	pod.Spec.Host = hostname
 
-	// Currently just simply follow the same format in resthandler.go
+	// TODO: Change this to use latest.Version once GCE containerVM supports v1beta3.
```

Comment:
Why do we need to wait? If we cut a new VM image that contained this change, it would also include v1beta3 support.